routes/project/inference_endpoint.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class Clover
+  hash_branch(:project_prefix, "inference-endpoint") do |r|
+    unless @project.get_ff_inference_ui
+      response.status = 404
+      r.halt
+    end
```

Comment:
@jeremyevans special 😄 
```suggestion
    next unless @project.get_ff_inference_ui
```

Reply:
neat